db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
```

Comment:
Right now I don't see any use of Id in audit table, will remove it and keep state_machine_instance_id, state_id and retry number combination as key.